### PR TITLE
Prevent browsers from writing sensitive form inputs to user data

### PR DESCRIPTION
**Pull request by lukaw3d**
Related to https://github.com/oasisprotocol/oasis-wallet-web/pull/1171

### CODE REVIEW

File: src/popup/pages/ImportAccount/index.js [hunk 1]
```diff
@@ -124,15 +126,20 @@ class ImportAccount extends React.Component {
...
         history={this.props.history}>
-        <div className="import-account-container">
-          {this.renderInput()}
-        </div>
-        {this.renderBottom()}
+        <form onSubmit={this.onSubmit} autoComplete="off">
+          <div className="import-account-container">
+            {this.renderInput()}
+          </div>
+          {this.renderBottom()}
```
Comment (csillag) [inline src/popup/pages/ImportAccount/index.js:141]: Shouldn't we move the actual submit code out from the button click handler, into the form submit handler?

Comment (csillag) [reply src/popup/pages/ImportAccount/index.js:141]: Oh, I see it now.

**Review by csillag — APPROVED** [review]
Reviewed. LGTM.

**Review by pro-wh — APPROVED** [review]
implemented as described

this also does part of the release process

**Pull request merged**